.github/workflows/build-rust-code.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
   build-rust:
     runs-on: ubuntu-latest
-    timeout-minutes: 40
+    timeout-minutes: 50
```

Comment:
its a shame that the cache limit is 5gb (https://github.com/actions/cache/tree/master) ... seems like we could speed up a lot of our builds if we cached the deps